decidim-admin/app/packs/stylesheets/decidim/admin/_tribute.scss

```diff
@@ -0,0 +1,40 @@
+.tribute-container {
+  @apply w-full relative;
+
+  #results {
+    @apply absolute max-h-[225px] top-full left-0 right-0 p-0 my-2 rounded z-50 overflow-y-scroll bg-white;
+
+    border: 1px solid rgba(33, 33, 33, 0.1);
```

Comment:
@Crashillo we should be using a color defined in the tailwind.config.js, right? Can you review other appearances?